src/store/actions/market/index.js

```diff
@@ -2,3 +2,4 @@ export * from './addMarkets'
 export * from './fetchMarkets'
 export { default as addMarkets } from './addMarkets'
 export { default as fetchMarkets } from './fetchMarkets'
+export * from './old'
```

Comment:
What's that? 😮

Reply:
this is the old `market.js` file from actions/market. When we moved them together, `market.js` and `markets/` became conflicted. I moved the old market.js into this file and export them all from here. We should get rid fo them over time and move them into single files.